index.html

```diff
@@ -629,7 +629,101 @@ <h3>Presentations</h3>
 that are often, but not required to be, related.
         </p>
 
-      </section>
+    </section>
+    <section>
+      <h3>Lifecycle</h3>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+This section seeks to illuminate the lifecycle of a credential. Credentials 
```

Comment:
I'd recommend changing this to "This section illustrates the lifecycle of a credential."